doit.go

```diff
@@ -139,6 +139,37 @@ func (v Version) Complete(lv LatestVersioner) string {
 	return buffer.String()
 }
 
+// CompleteJSON is the complete version for doit, formatted as JSON.
+func (v Version) CompleteJSON(lv LatestVersioner) string {
+	versionInfo := &struct {
+		Version      string `json:"version,omitempty"`
+		Commit       string `json:"commit,omitempty"`
+		Notification string `json:"notification,omitempty"`
+	}{
+		Version: fmt.Sprintf("doctl version %s", v.String()),
+	}
+
+	if v.Build != "" {
+		versionInfo.Commit = fmt.Sprintf("\nGit commit hash: %s", v.Build)
```

Comment:
should this just be `v.Build`?

Reply:
Yeah, it probably should, I just started by copying this verbatim from the regular output, and then sticking it in fields that roughly made sense. Definitely don't want a newline in there though haha